cedar-lean/Cedar/Thm/Authorization/Evaluator.lean

```diff
@@ -57,16 +57,15 @@ theorem and_true_implies_right_true {e₁ e₂ : Expr} {request : Request} {enti
   simp [evaluate, h₂, Result.as, Coe.coe, Value.asBool] at h₁
   generalize h₃ : (evaluate e₂ request entities) = r₂
   simp [h₃] at h₁
-  cases r₂ <;> simp [Lean.Internal.coeM] at h₁
```

Comment:
It is so nice to see these go away!

Reply:
Pretty sure Lean 4.6 or 4.7 newly enabled omitting the `Lean.Internal.coeM` and friends

Reply:
Yes, it was the update for the win!